generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js

```diff
@@ -1,162 +1,170 @@
-'use strict';
+(function() {
+    'use strict';
 
-angular.module('<%=angularAppName%>')
-    .config(function ($stateProvider) {
-        $stateProvider
-            .state('<%= entityStateName %>', {
-                parent: 'entity',
-                url: '/<%= entityUrl %><% if (pagination == 'pagination' || pagination == 'pager') { %>?page&sort&search<% } %>',
-                data: {
-                    authorities: ['ROLE_USER'],
-                    pageTitle: <% if (enableTranslation){ %>'<%= angularAppName %>.<%= entityTranslationKey %>.home.title'<% }else{ %>'<%= entityClassPlural %>'<% } %>
-                },
-                views: {
-                    'content@': {
-                        templateUrl: 'app/entities/<%= entityFolderName %>/<%= entityFileName %>.html',
-                        controller: '<%= entityClass %>ManagementController'
-                    }
-                },
-                <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
-                params: {
-                    page: {
-                        value: '1',
-                        squash: true
+    angular.module('<%=angularAppName%>')
+        .config(function ($stateProvider) {
+            $stateProvider
+                .state('<%= entityStateName %>', {
+                    parent: 'entity',
+                    url: '/<%= entityUrl %><% if (pagination == 'pagination' || pagination == 'pager') { %>?page&sort&search<% } %>',
+                    data: {
+                        authorities: ['ROLE_USER'],
+                        pageTitle: <% if (enableTranslation){ %>'<%= angularAppName %>.<%= entityTranslationKey %>.home.title'<% }else{ %>'<%= entityClassPlural %>'<% } %>
+                    },
+                    views: {
+                        'content@': {
+                            templateUrl: 'app/entities/<%= entityFolderName %>/<%= entityFileName %>.html',
+                            controller: '<%= entityClass %>ManagementController',
+                            controllerAs: 'vm'
+                        }
                     },
-                    sort: {
-                        value: 'id,asc',
-                        squash: true
+                    <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
+                    params: {
```

Comment:
the diff shows more changes, but its correct compared to latest master so this is fine